third_party/move/move-vm/runtime/src/storage/dummy.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    loader::{Module, Script},
+    storage::{module_storage::ModuleStorage, script_storage::ScriptStorage, verifier::Verifier},
+};
+use move_binary_format::{
+    errors::{PartialVMError, PartialVMResult},
+    file_format::CompiledScript,
+    CompiledModule,
+};
+use move_core_types::{
+    account_address::AccountAddress, identifier::IdentStr, metadata::Metadata,
+    vm_status::StatusCode,
+};
+use std::sync::Arc;
+
+macro_rules! not_implemented {
```

Comment:
given that rust standard macro that panics for this is unimplemented, and the difference here is twofold:
(a) error instead of panic, and
(b) no custom error message
let's name it smt like unimplemented_code_cache_error or smt along those lines?

Reply:
done